news/constants/categories.ts

```diff
@@ -9,11 +9,11 @@ export const supportedCategories = [
     ],
   },
   { category: 'Business', subCategories: ['Economy', 'Finance', 'Industrial'] },
-  { category: 'Tech', subCategories: ['Internet', 'Media', 'Social Media'] },

```

Comment:
Please check if there is any Tech article with Social Media as sub category. If yes, please update that to the same in dynamodb.